libchisel/src/dropsection.rs

```diff
@@ -29,6 +32,34 @@ impl<'a> ChiselModule<'a> for DropSection {
     }
 }
 
+impl From<std::num::ParseIntError> for ModuleError {
+    fn from(error: std::num::ParseIntError) -> Self {
+        ModuleError::Custom(error.description().to_string())
+    }
+}
+
+impl ModuleConfig for DropSection {
+    fn with_defaults() -> Result<Self, ModuleError> {
+        Err(ModuleError::NotSupported)
+    }
+
+    fn with_config(config: &HashMap<String, String>) -> Result<Self, ModuleError> {
+        for (key, val) in config.iter() {
```

Comment:
Why do we need the loop here if it will always return on first iteration?

Reply:
Legacy. I guess it could pick a random key/val and do it from there.

Reply:
That would result in some strange behavior, I feel.
Perhaps enforcing mutual exclusion of dropsection modes here would make the best sense.

Reply:
Yeah I'm torn on that one. Enforcing that will add a ton of complexity – that's how I started first.